### PR TITLE
[QPPA-1306] Add programName and practiceId to docs

### PR DESCRIPTION
**Pull request by kalvinwang**
This PR adds programName and practiceId to the docs. Functionality implemented in https://github.com/CMSgov/qpp-submissions-api/pull/737.

Because these are both optional fields unlikely to be used by most submitters, I am not modifying the tutorials or the example json/xml to include them, since those reasonably only include the necessary, required fields.

Reviewer: @marimiyachi

### CODE REVIEW

File: src/components/api-reference/schemas/measurement-sets.js [hunk 1]
```diff
@@ -12,9 +12,11 @@ const FIELDS = [
...
+  {name: 'practiceId', value: 'string', description: 'The ID of the practice associated with the measurement set. Required if programName is set to "cpcPlus". Must be omitted if programName is not set to "cpcPlus".', notes: 'writable, optional'},
   {name: 'performanceStart', value: 'string', description: 'A date in RFC 3339 format with only the date part (for instance, "2013-01-15"). The first date when the measurement data is applicable.', notes: 'writable'},
   {name: 'performanceEnd', value: 'string', description: 'A date in RFC 3339 format with only the date part (for instance, "2013-01-15"). The last date when the measurement data is applicable.', notes: 'writable'},
-  {name: 'measureSet', value: 'string', description: 'The speciality set which the measurement set is attesting. Acceptable values are "generalSurgery" or "other"', notes: 'writable, optional'},
+  {name: 'measureSet', value: 'string', description: 'The speciality set which the measurement set is attesting. Acceptable values are <b>"generalSurgery"</b> or <b>"other"</b>', notes: 'writable, optional'},
```
Comment (marimiyachi) [inline src/components/api-reference/schemas/measurement-sets.js]: Out of scope of this ticket but this is outdated...we accept a whole slew of measure sets now!

Comment (kalvinwang) [reply src/components/api-reference/schemas/measurement-sets.js]: wow yeah! I can add them into this pr, seems reasonable.

allergyImmunology, anesthesiology, cardiology, dermatology, diagnosticRadiology, electrophysiologyCardiacSpecialist, emergencyMedicine, gastroenterology, generalOncology, generalPracticeFamilyMedicine, generalSurgery, hospitalists, internalMedicine, interventionalRadiology, mentalBehavioralHealth, neurology, obstetricsGynecology, ophthalmology, orthopedicSurgery, otolaryngology, pathology, pediatrics, physicalMedicine, plasticSurgery, preventiveMedicine, radiationOncology, rheumatology, thoracicSurgery, urology, vascularSurgery

Comment (kalvinwang) [reply src/components/api-reference/schemas/measurement-sets.js]: @marimiyachi can you just confirm "other" is no longer an option?

Comment (marimiyachi) [reply src/components/api-reference/schemas/measurement-sets.js]: Yup - other is not an option.

Comment (kalvinwang) [reply src/components/api-reference/schemas/measurement-sets.js]: thanks, added

**Comment by kalvinwang** [comment]
I'll merge this PR to master and deploy once [the main PR](https://github.com/CMSgov/qpp-submissions-api/pull/737) has been deployed to production (otherwise the docs will be inaccurate)

**Pull request merged**